CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 2.9.4 (Unreleased)
 
+Features:
```

Comment:
needs to be all uppercase to stay consistent?

Reply:
Fixed it.